scripts/update_config.py

```diff
@@ -0,0 +1,427 @@
+import argparse
+import json
+import os
+import re
+from collections import defaultdict
+
+import boto3
+import yaml
+
+MAPPING = {
+    "agents": ["cybench", "gaia", "gdm", "piqa", "swe_bench"],
+    "assistants": ["gaia", "piqa"],
+    "coding": ["cybench", "humaneval", "mbpp", "osworld", "swe_bench"],
+    "cybersecurity": [
+        "cybench",
+        "cyse2_vulnerability_exploit",
+        "cyse2_interpreter_abuse",
+        "cyse2_prompt_injection",
+        "gdm",
+    ],
+    "knowledge": ["commonsense_qa", "gpqa_diamond", "mmlu_pro"],
+    "mathematics": ["gsm8k", "mathvista"],
+    "multimodal": ["mathvista", "mmmu_open", "mmmu_multiple_choice"],
+    "reasoning": [
+        "bbh",
+        "hellaswag",
+        "ifeval",
+        "mmmu_open",
+        "mmmu_multiple_choice",
+        "mmlu_pro",
+    ],
+    "safeguards": ["agentharm"],
+}
+
+DASHBOARD_LOG_FILE_SUFFIX = ".dashboard.json"
+
+ENV_ORDER = ["prod", "stage", "dev", "test"]
+
+BUCKET_NAME = os.environ.get("AWS_S3_BUCKET", "inspect-evals-dashboard")
+
+
+def parse_paths():
+    # Get paths directly from S3
+    s3 = boto3.client("s3")
+    paginator = s3.get_paginator("list_objects_v2")
+    result = []
+
+    for page in paginator.paginate(Bucket=BUCKET_NAME, Prefix="logs/stage/"):
```

Comment:
why does this only look at staging logs?

Reply:
Fixed!